src/array_api_stubs/_draft/manipulation_functions.py

```diff
@@ -42,14 +42,14 @@ def broadcast_to(x: array, /, shape: Tuple[int, ...]) -> array:
     Parameters
     ----------
     x: array
-        array to broadcast.
+        array to broadcast. Must be capable of being broadcast to the specified ``shape`` (see :ref:`broadcasting`). If the array is incompatible with the specified shape, the function should raise an exception.
```

Comment:
Sorry never looked it, but a small nit: it should be "must raise an error".

Reply:
Opened https://github.com/data-apis/array-api/pull/897 to resolve.